api/src/wfl/server.clj

```diff
@@ -96,7 +93,8 @@
                   (catch UserException e
                     (log/warn (format "Error updating workload %s" uuid))
                     (log/warn e)
-                    (notify-watchers watchers uuid e))))))
+                    ;; need to deserialize the watchers array
+                    (slack/notify-watchers (map read-string watchers) uuid e))))))
```

Comment:
I think this ``read-string` should be moved into the `load-workload-impl` so that we don't need to deserialise elsewhere.

Reply:
that's a good idea, will do!